src/Spectre.libPlatform/Math.h

```diff
@@ -351,4 +352,34 @@ constexpr std::vector<DataType> abs(gsl::span<const DataType> data)
     static_assert(std::is_arithmetic_v<DataType>, "DataType: expected arithmetic.");
     return Functional::transform(data, [](DataType entry) { return std::abs(entry); });
 }
+
+/// <summary>
+/// transform to opposite values
+/// </summary>
+/// <param name="data">The data.</param>
+/// <returns>Vector consisting of opposite values of elements of first</returns>
+template <class DataType>
+constexpr std::vector<DataType> negate(gsl::span<const DataType> data)
```

Comment:
This one may be usefuly, but it won't work for `std::vector<bool>`, so please be aware of this.

Reply:
done